kolibri/core/assets/src/views/language-switcher/LanguageSwitcherList.vue

```diff
@@ -12,14 +12,15 @@
       />
     </UiIconButton>
 
-    <span class="selected">
-      {{ selectedLanguage }}
+    <span class="selected" :title="selectedLanguage.english_name">
```

Comment:
I am not a fan of this, why english?

Reply:
It stemmed from this conversation https://github.com/learningequality/clearinghouse/issues/274

I agree that really every language name should be translated into every other language, and the tooltips should be localized the same as all other application text. However, I think there's also value and not much harm in adding English tooltips now and translating them in the next release.

Reply:
(cc @radinamatic)

Reply:
:okay:

Reply:
created a follow-up issue for this: #4483